modules/nf-core/mixcr/analyze/tests/main.nf.test

```diff
@@ -0,0 +1,67 @@
+nextflow_process {
+
+    name "Test Process MIXCR_ANALYZE"
+    script "../main.nf"
+    process "MIXCR_ANALYZE"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "mixcr"
+    tag "mixcr/analyze"
+
+    test("10xgenomics vdj - fastq") {
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    [ file(params.modules_testdata_base_path + 'genomics/homo_sapiens/10xgenomics/cellranger/5k_cmvpos_tcells/fastqs/vdj/subsampled_5k_human_antiCMV_T_TBNK_connect_VDJ_S1_L001_R1_001.fastq.gz', checkIfExists: true), file(params.modules_testdata_base_path + '/genomics/homo_sapiens/10xgenomics/cellranger/5k_cmvpos_tcells/fastqs/vdj/subsampled_5k_human_antiCMV_T_TBNK_connect_VDJ_S1_L001_R2_001.fastq.gz', checkIfExists: true) ]
+                ]
+                input[1] = "10x-vdj-tcr"
+                input[2] = "hsa"
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                        process.out.clones,
+                        // process.out.vdjca,
+                        // process.out.clns,
+                        process.out.versions
```

Comment:
```suggestion
                        process.out.versions,
                        path(process.out.versions[0]).yaml
```
This gives a nice human readable check that the versions output is printing correctly.